spec.md

```diff
@@ -1150,11 +1155,12 @@ If the plugin is unable to complete the NodeUnpublishVolume call successfully, i
 If the conditions defined below are encountered, the plugin MUST return the specified gRPC error code.
 The CO MUST implement the specified error recovery behavior when it encounters the gRPC error code.
 
-Condition | gRPC Code | Description | Recovery Behavior
-| --- | --- | --- | --- |
+| Condition | gRPC Code | Description | Recovery Behavior |
+|-----------|-----------|-------------|-------------------|
 | Volume does not exists | 5 NOT_FOUND | Indicates that a volume corresponding to the specified `volume_id` does not exist. | Caller MUST verify that the `volume_id` is correct and that the volume is accessible and has not been deleted before retrying with exponential back off. |
```

Comment:
NodeUnpublishVolume should respond `0 OK` if the volume is not mounted at the `target_path`.

Reply:
This is slightly different than the `NOT_FOUND` case, where the volume does not exist. I'll add some wording at the top to say that if the volume published at `target_path` has already been unpublished, the plugin should return OK.